content/authors/neha0812/index.md

```diff
@@ -0,0 +1,47 @@
+---
+# Display name
+title: <Neha Sharma>
+
+# Username (this should match your github username)
+authors:
+- neha0812
+
+superuser: false
+
+# Role/position e.g. Graduate student
+role: PhD Student
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name: University of Alberta
+  url: https://www.ualberta.ca/index.html
+
+# A list of your interests, add as many as you want
+interests:
+- Mathematical Finance
+- Stochastic Analysis
+- Machine Leaning
+-Data Science
```

Comment:
YAML is very sensitive to white space, can you add a space between the `-` and the word `Data` e.g.
```
- Data Science
```

Reply:
I have changed the .jpg file from avator to avatar. Thanks